src/main/java/fish/payara/extras/upgrade/BaseUpgradeCommand.java

```diff
@@ -204,13 +203,6 @@ protected void reinstallNodes() throws IOException, CommandException, Configurat
             // added via the modules directory
             ServiceLocator serviceLocator = createServiceLocator();
             ConfigParser parser = new ConfigParser(serviceLocator);
-            try {
-                parser.logUnrecognisedElements(false);
-            } catch (NoSuchMethodError noSuchMethodError) {
-                logger.log(Level.FINE,
-                        "Using a version of ConfigParser that does not support disabling log messages via method",
-                        noSuchMethodError);
-            }
```

Comment:
Do we not need this anymore?
This was to prevent HK2 from spamming the log with meaningless info during the node upgrade process.

Reply:
done